### PR TITLE
Fix minor typos

### PR DESCRIPTION
**Pull request by phil-davis**
Typo changes are only in docs, code comments, and tests. So should be no run-time effect here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Better Reflection is a reflection API that aims to improve and provide more feat
 * Fetch return type declaration and parameter type declarations in PHP 7 code
 * Change or remove PHP 7 parameter type and return type declarations from methods and functions
 * Change the body of a function or method to do something different
-* *Moar stuff coming soon!*
```
Comment (Ocramius) [inline README.md:23]: This was intentional :D

File: test/core/ReflectionClass_isIterateable_001.phpt [hunk 2]
```diff
@@ -16,18 +16,18 @@ Class IteratorImpl implements Iterator {
 	public function current() {}
 	public function valid() {}
 }
-Class IterarorAggregateImpl implements IteratorAggregate {
+Class IteratorAggregateImpl implements IteratorAggregate {
```
Comment (Ocramius) [inline test/core/ReflectionClass_isIterateable_001.phpt:19]: This stuff is copied directly from php-src and should not be touched

File: test/core/bug64936.phpt [hunk 3]
```diff
@@ -4,7 +4,7 @@ ReflectionMethod::getDocComment() uses left over doc comment from previous scann
 opcache.save_comments=1
 --SKIPIF--
 <?php
-if (!extension_loaded('reflection') || !extension_loaded('tokenizer')) print 'skip missing reflection of tokernizer extension';
+if (!extension_loaded('reflection') || !extension_loaded('tokenizer')) print 'skip missing reflection of tokenizer extension';
```
Comment (Ocramius) [inline test/core/bug64936.phpt:7]: This is copied directly from php-src and should not be touched

File: README.md (hunk 1 above)
Comment (phil-davis) [reply README.md:23]: OK - someone's name?

Comment (Ocramius) [reply README.md:23]: Just joking slang for "more" :)

**Comment by phil-davis** [comment]
Reverted the bits that needed to stay as-is.

**Pull request merged**